tests/scripts/build_dgl.sh

```diff
@@ -9,6 +9,12 @@ fi
 
 if [[ $1 != "cpu" ]]; then
     CMAKE_VARS="$CMAKE_VARS -DUSE_CUDA=ON"
+    if [[ $1 == "gpu" ]]; then
```

Comment:
I think the if statement here is not needed. The code inside the if can be run for cugraph test without any problem.